benchmarks/test_ptpk_lpt.py

```diff
@@ -12,6 +12,12 @@
                       T_CMB=2.725,
                       transfer_function='boltzmann_camb')
 
+# The accuracy of the CCL-internal sigmaR computation isn't that high
+SIGMA8_TOLERANCE = np.sqrt(
+    COSMO._accuracy_params["INTEGRATION_SIGMAR_EPSREL"]
+    * np.log(10)/(2*np.pi**2)
+)
+
```

Comment:
What is this number roughly?

Reply:
I'm kind of confused by why we now need this everywhere.

Reply:
After some digging, I think the CAMB estimate of sigma8 is quite rough, which was causing these issues. I think I can rewrite the CAMB interface to use the CCL sigma8 estimate though, which should resolve all these test accuracy problems.